python/kserve/kserve/protocol/rest/server.py

```diff
@@ -0,0 +1,177 @@
+                FastAPIRoute(r"/v2/health/ready", v2_endpoints.ready,
+                             response_model=ServerReadyResponse, tags=["V2"]),
+                FastAPIRoute(r"/v2/models/{model_name}",
+                             v2_endpoints.model_metadata, response_model=ModelMetadataResponse, tags=["V2"]),
+                FastAPIRoute(r"/v2/models/{model_name}/versions/{model_version}",
+                             v2_endpoints.model_metadata, tags=["V2"], include_in_schema=False),
+                FastAPIRoute(r"/v2/models/{model_name}/ready",
+                             v2_endpoints.model_ready, response_model=ModelReadyResponse, tags=["V2"]),
+                FastAPIRoute(r"v2/models/{model_name}/versions/{model_version}/ready",
+                             v2_endpoints.model_ready, response_model=ModelReadyResponse, tags=["V2"]),
+                FastAPIRoute(r"/v2/models/{model_name}/infer",
+                             v2_endpoints.infer, methods=["POST"], response_model=InferenceResponse, tags=["V2"]),
+                FastAPIRoute(r"/v2/models/{model_name}/versions/{model_version}/infer",
+                             v2_endpoints.infer, methods=["POST"], tags=["V2"], include_in_schema=False),
+                FastAPIRoute(r"/v2/repository/models/{model_name}/load",
+                             v2_endpoints.load, methods=["POST"], tags=["V2"]),
+                FastAPIRoute(r"/v2/repository/models/{model_name}/unload",
+                             v2_endpoints.unload, methods=["POST"], tags=["V2"]),
+            ], exception_handlers={
+                InvalidInput: invalid_input_handler,
+                InferenceError: inference_error_handler,
+                ModelNotFound: model_not_found_handler,
+                ModelNotReady: model_not_ready_handler,
+                NotImplementedError: not_implemented_error_handler,
+                Exception: generic_exception_handler
+            }
+        )
+
+
+class UvicornProcess(multiprocessing.Process):
+
+    def __init__(self, http_port: int, sockets: List[socket.socket],
+                 data_plane: DataPlane, model_repository_extension, enable_docs_url):
+        super().__init__()
+        self.sockets = sockets
+        self.server = RESTServer(data_plane, model_repository_extension, enable_docs_url)
+        app = self.server.create_application()
+        app.add_middleware(
+            TimingMiddleware,
+            client=PrintTimings(),
+            metric_namer=StarletteScopeToName(prefix="kserve.io", starlette_app=app)
+        )
+        self.cfg = uvicorn.Config(
+            app=app,
+            host="0.0.0.0",
+            port=http_port,
+            log_config={
+                "version": 1,
+                "formatters": {
+                    "default": {
+                        "()": "uvicorn.logging.DefaultFormatter",
+                        "datefmt": "%Y-%m-%d %H:%M:%S",
```

Comment:
instead of hard coding this twice save it as a constant? not rlly a big deal if we dont just maybe better consistency if we want to keep them the same